front/lib/workspace_usage.ts

```diff
@@ -432,13 +427,29 @@ export async function getAgentUsageData(
 
 function generateCsvFromQueryResult(
   rows:
+    | WorkspaceUsageQueryResult[]
     | userUsageQueryResult[]
     | AgentUsageQueryResult[]
     | MessageUsageQueryResult[]
     | builderUsageQueryResult[]
 ) {
   const csvHeader = Object.keys(rows[0]).join(",") + "\n";
-  const csvContent = rows.map((row) => Object.values(row).join(",")).join("\n");
+  const csvContent = rows
+    .map((row) =>
+      Object.values(row)
+        .map((value) => {
+          // Escape quotes and wrap in quotes if the value contains commas or quotes
```

Comment:
Escaping quotes ourselves is where I draw the limit :D

Let's use this: https://github.com/dust-tt/dust/blob/30404fa506baddafd56df3bd36f1c16caf63697f/front/components/actions/tables_query/TablesQueryActionDetails.tsx#L3